docs/config_search.md

```diff
@@ -108,29 +156,20 @@ profile_models:
           count: [1, 2]
 ```
 
-#### Important Note about Remote Mode
+_In this example, MA will not sweep through any values, and will only try the concurrencies listed below:_
 
-In the remote mode, `model_config_parameters` are always ignored because Model
-Analyzer has no way of accessing the model repository of the remote Triton
-Server. In this mode, only concurrency values can be swept.
-
-### Manual Brute Search
-
-In addition to the automatic config search, Model Analyzer supports a manual
-config search mode. To enable this mode, you can specify `model_config_parameters`
-to sweep through, or set `--run-config-search-disable`
+```yaml
+model_repository: /path/to/model/repository/
 
-Unlike automatic mode, this mode is not limited to `max_batch_size`, `dynamic_batching`, and `instance_count` config parameters. Using manual
-config search, you can create custom sweeps for every parameter that can be
-specified in the model configuration. Model Analyzer only checks the syntax
-of the `model_config_parameters` that is specified and cannot guarantee that
-the configuration that is generated is loadable by Triton.
+profile_models:
+  model_1:
+    parameters:
+      concurrency: 1,2,3,128
+```
 
-You can also specify `concurrency` ranges to sweep through. If unspecified, it will
-automatically sweep concurrency for every model configuration (unless `--run-config-search-disable`
-is set, in which case it will only use the concurrency value of 1)
+_This example shows how a value other than instance, batch size or concurrency can be swept, and will sweep through eight configs:<br>_
 
-An example Model Analyzer Config that performs manual sweeping looks like below:
+- \[2 _ `max_batch_size`] \* \[2 _ `max_queue_delay_microseconds`] \* \[2 \* `instance_group`].
```

Comment:
underscores should be asterisks?  Or shouldn't exist at all?

Reply:
Forgot the backslash and vscode converted the asterisks to underscores. Fixed